sdk/core/core-paging/review/core-paging.api.md

```diff
@@ -15,13 +15,14 @@ export interface PagedAsyncIterableIterator<T, PageT = T[], PageSettingsT = Page
 }
 
 // @public
-export interface PagedResult<TPage, TPageSettings = PageSettings, TLink = string> {
+export interface PagedResult<TPage, TPageSettings = PageSettings, TLink = string, T = unknown> {
```

Comment:
nitpick: Should `T` be renamed to a more descriptive name, e.g. `TElements`?

Reply:
+1 for `TElement` since I feel like if we rewrote `PagedAsyncedIterableIterator` today we wouldn't use plain `T` -- @bterlson can tell me I'm wrong though 😉

Reply:
I would still use T in some cases (e.g. Array<TElement> is a little awkward) but yeah generally TSomething is best, and especially when there are multiple type params.

Reply:
I will implement the name changes with this PR instead of opening a new one so we can have a consistent generics name for core-paging